2020-11-25 04:08:18,067: DEBUG: NEW RUN: Client, TEST: Write_Multiple_Clients_Five_Threads5
2020-11-25 04:08:18,067: DEBUG: The client is starting.
2020-11-25 04:08:18,067: DEBUG: The client is starting on port 15687
2020-11-25 04:08:18,067: DEBUG: Sent WRQ: b'\x00\x02shutdown.txt\x00netascii\x00'
2020-11-25 04:08:18,068: DEBUG: File exists, can send it
2020-11-25 04:08:18,069: DEBUG: First packet received. Raw bytes: b'\x00\x05\x00\x02filename already exists at destination\x00'
2020-11-25 04:08:18,069: DEBUG: Sending: shutdown.txt
2020-11-25 04:08:18,069: DEBUG: New packet from: ('127.0.0.1', 58559), contents: b'\x00\x05\x00\x02filename already exists at destination\x00'
2020-11-25 04:08:18,089: ERROR: Shutting down due to error packet. Error code: 2, error message: b'filename already exists at destination'
2020-11-25 04:08:18,089: WARNING: File didnt close on its own, closing now
2020-11-25 04:08:18,169: DEBUG: Client finished.
2020-11-25 04:08:18,170: DEBUG: Closing Socket
